nuitka/MainControl.py

```diff
@@ -937,12 +940,23 @@ def main():
     addIncludedDataFilesFromFileOptions()
     addIncludedDataFilesFromPackageOptions()
 
+    if Options.options.profile_optimization:
```

Comment:
This feels like it wants to be a context manager, where this is done, potentially also delaying the imports to there inside a decision, if that is going to do anything at all.